arroyo-sql-testing/Cargo.toml

```diff
@@ -4,6 +4,7 @@ version = "0.7.0"
 edition = "2021"
 
 [features]
+default = ["integration-tests"]
```

Comment:
is this intentional?

Reply:
No, I just do that to get VS Code to resolve the feature.